app/views/events/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<h3 class="text-center mt-4 text-white">
+<span><i class="fas fa-forward"></i></span>
+Upcoming Events
+</h3>
+
+<table class="table w-75 m-auto">
+    <thead class="thead-dark">
+        <tr>
+          <th>Status</th>
+            <th>Description</th>
+            <th>Creator</th>
+            <th>Date</th>
+            <th>Location</th>
+            <th class="text-center">Show Event Page</th>
+        </tr>
+    </thead>
+    <tbody>
+      <% current_user.attended_events.upcoming_events.each do |event| %>
```

Comment:
- [ ] Event's index page displays ALL Events separated into past and upcoming events -- You are displaying only current_user attended_events

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png)